terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -0,0 +1,1261 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-qe/job/manager-5.0-qe-build-validation-NUE"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = string
+  default = "export PRODUCT='SUSE-Manager' && run-testsuite"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = string
+  default = "https://github.com/uyuni-project/uyuni.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = string
+  default = "master"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = string
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = string
+  default = "Results HEAD Build Validation $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
```

Comment:
I think here and in the other e-mails related variables we should refer to 5.0, not HEAD.

Reply:
Fixed. Thank you, Paolo.